sdk/communication/communication-network-traversal/review/communication-network-traversal.api.md

```diff
@@ -25,7 +25,10 @@ export class CommunicationRelayClient {
     constructor(endpoint: string, credential: TokenCredential, options?: CommunicationRelayClientOptions);
     getRelayConfiguration(options?: GetRelayConfigurationOptions): Promise<CommunicationRelayConfiguration>;
     getRelayConfiguration(routeType: RouteType, options?: GetRelayConfigurationOptions): Promise<CommunicationRelayConfiguration>;
-    getRelayConfiguration(user: CommunicationUserIdentifier, routeType?: RouteType, options?: GetRelayConfigurationOptions): Promise<CommunicationRelayConfiguration>;
+    getRelayConfiguration(ttl: number, options?: GetRelayConfigurationOptions): Promise<CommunicationRelayConfiguration>;
```

Comment:
I wonder if `ttl` can be an option in `GetRelayConfigurationOptions` instead of a parameter of the operation?

Reply:
I agree, all optional arguments should go into the options bag. If I see this right we should then only have one function
```typescript
getRelayConfiguration(options?: GetRelayConfigurationOptions): Promise<CommunicationRelayConfiguration>;
```
and `user`, `ttl`, and `routeType` should all be optional properties inside the options.